app/main.py

```diff
@@ -2,0 +1,9 @@
+def copy_file(command: str) -> None:
+    command_list = command.split(" ")
```

Comment:
You are using `split` method without any parameters, which splits by any whitespace. This can cause errors if the command contains file paths with spaces. It's better to use `split(' ', 2)` to split the command into exactly 3 parts: the command name, the source file path, and the destination file path.